Detectors/MUON/MCH/Simulation/src/Response.cxx

```diff
@@ -89,3 +89,38 @@ uint32_t Response::nSamples(float charge) const
   double signalParam[3] = {14., 13., 1.5};
   return std::round(std::pow(charge / signalParam[1], 1. / signalParam[2]) + signalParam[0]);
 }
+//_____________________________________________________________________
+float Response::eLossRatio(float logbetagamma) const
+{
+  // Ratio of particle mean eloss with respect MIP's Khalil Boudjemline, sep 2003, PhD.Thesis and Particle Data Book
+  /// copied from aliroot AliMUONv1.cxx
+  float eLossRatioParam[5] = {1.02138, -9.54149e-02, +7.83433e-02, -9.98208e-03, +3.83279e-04};
+  return eLossRatioParam[0] + eLossRatioParam[1] * logbetagamma + eLossRatioParam[2] * std::pow(logbetagamma, 2) + eLossRatioParam[3] * std::pow(logbetagamma, 3) + eLossRatioParam[4] * std::pow(logbetagamma, 4);
```

Comment:
Unfortunately `std::pow()` is a function operating on double values. So this will utilize multiple very expensive function calls.
Would you be able to change the code so that we use only multiplications:
```
// for instance
lbetagamma2 = logbetagamma*logbetagamma
lbetagamma3 = lbetagamma2 * logbetagamma
lbetagamma4 = lbetagamma2 * lbetagamma2
```
Otherwise I can do it in a separate commit.